sphere_tesselate.f90

```diff
@@ -899,6 +918,436 @@ function tolower(a) result(b)
          b(i:i) = achar(iachar(a(i:i)) + 32)
    enddo
 end function tolower
+
+
+!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!========================*****=============================*****================
+!!! Jack Walpole routines
+!========================*****=============================*****================
+!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+!===============================================================================
+subroutine st_neighbours_around_point(pt_id,tess,neighbours)
+!===============================================================================
+! DEAD CODE
+! This routine gets cripplingly slow when mesh level gets beyond about 4 or 5.
+
+! input argument pt_id is index of point 
+! input argument t is tesselation
+! output local_tess is tesselation containing all triangles connected to point p
+   integer, intent(in)            :: pt_id
+   type(tesselation), intent(in)  :: tess
+   integer, intent(out), allocatable, dimension(:) :: neighbours
+   integer, allocatable, dimension(:) :: tmp_nbrs
+   integer                        :: n_nbrs, count, ii, jj, kk
+   type(point)                    :: pt
+   type(triangle)                 :: tr  
+  
+   ! test
+   pt = tess%p(pt_id)
+!    write (*,*) 'Number of points:',tess%np
+!    write (*,*) 'Point:', pt_id
+!    write (*,*) pt
+   
+   ! if pt_id is less than or equal to 12 then this is a "special" point
+   ! corresponding to the original octahedron.  It is special because it 
+   ! has only 5 neighbours (all other points have 6 neighbours).
+   if ( pt_id <= 12 ) then
```

Comment:
It's always good to match the style of the codebase—note it doesn't have spaces within brackets, for example.